zboxcore/sdk/consensus.go

```diff
@@ -3,7 +3,7 @@ package sdk
 import "sync"
 
 type Consensus struct {
-	sync.RWMutex
+	*sync.RWMutex
```

Comment:
Pointer mutex need to be initialized manually in all the places where Consensus is defined. Please check and fix, otherwise we will see panics

Reply:
thanks, intialized all instances of Mutex